source/index.md

```diff
@@ -30,12 +30,16 @@ This documentation supplements the official [Nix](http://nixos.org/nix/manual/),
 and [NixOps](http://nixos.org/nixops/manual/) manuals.
 
 ```{toctree}
-:glob: true
+:glob:
+:caption: Tutorials
 :maxdepth: 1
+:hidden:
 
-tutorials/index.md
-templates/index.md
-anti-patterns/index.md
+tutorials/install-nix.md
+tutorials/first-steps/index.md
+tutorials/nixos/*
```

Comment:
Doesn't need to be nested for now, since it only has one item inside. We can always add more levels later.